routes/section8/schema.js

```diff
@@ -0,0 +1,12 @@
+const Schema = {
+  firstname: {
+    isLength: {
+      errorMessage: 'errors.firstname.length',
+      options: { min: 3, max: 200 },
```

Comment:
Usually just safer not to validate names.
https://en.wikipedia.org/wiki/Cy_Young

Reply:
Okay just realized that was a nickname so I'm not as clever as I thought but I'm sure someone has a 2 letter name out there

Reply:
lol, there's no name field. This was the sample validation from the route template, should have just been removed... now it is.

Reply:
The error was due to missing the csrf token field. Added it, and have also submitted a PR to the node starter repo adding that to the route template.